nodes/test/audio_tts/test_iglobal.py

```
@@ -0,0 +1,131 @@
+        mock_engine.synthesize.return_value = {'path': fake_path, 'mime_type': 'audio/wav'}
+        mock_engine.config = dict(ig._config)
+        ig._engine = mock_engine
+
+        result = ig.synthesize('hello')
+        assert result['mime_type'] == 'audio/wav'
+        assert 'path' in result
+
+        try:
+            os.remove(fake_path)
+        except OSError:
+            pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)